Psyc/Assets/Scripts/FlashingLight.cs

```diff
@@ -0,0 +1,32 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class FlashingLight : MonoBehaviour
+{
+    public GameObject Light;
+    bool OnOff = true;
+    
+    IEnumerator flashing(float time) 
+    {
+        yield return new WaitForSeconds(time);
+        if (OnOff)
+        {
+            Light.SetActive(false);
+            OnOff = false;
+        }
+        else
+        {
+            Light.SetActive(true);
+            OnOff = true;
+        }
+    }
+
+    void Update()
+    {
+        float T = Random.Range(1, 1000);
```

Comment:
```suggestion
        var range = Random.Range(1, 1000);
```
temparatie varibele moet met var. De naam is niet duidelijk.